ci-info

```diff
@@ -244,6 +244,7 @@ elif [ -n "$CIRCLE_COMPARE_URL" ] ; then
 elif [ -n "$GITHUB_HEAD_REF" ] ; then
   # GitHub Actions pull request (no special handling is required for non-PR GitHub Actions jobs).
   CI_IS_PULL_REQUEST=true
+  CI_ORGANIZATION="${GITHUB_ACTOR}"
```

Comment:
`CI_ORGANIZATION` is set [earlier in this file](https://github.com/eisop-plume-lib/plume-scripts/blob/87d7811a459a3722e4d34cf753b884c3ac03919b/ci-info#L100).
It would seem nicer to do this up with the other CI organizations and check whether this is a GitHub Actions PR there.

Reply:
Thanks!